test/functional/sqlsrv/sqlsrv_client_info.phpt

```diff
@@ -15,7 +15,7 @@ var_dump( $client_info );
 --EXPECTREGEX--
 array\(4\) {
   \[\"(DriverDllName|DriverName)\"\]=>
-  (string\(15\) \"msodbcsql1[1-9].dll\"|string\(24\) \"libmsodbcsql-[1-9]{2}.[0-9].so.[0-9].[0-9]\")
+  (string\(15\) \"msodbcsql1[1-9].dll\"|string\(24\) \"libmsodbcsql-[1-9]{2}.[0-9].so.[0-9].[0-9]\"|string\(21\) \"libmsodbcsql.1[1-9].dylib\")
   \[\"DriverODBCVer\"\]=>
```

Comment:
dylib is only for mac isn't it?

Reply:
Yes.

Reply:
David we didn't wait for you to make such changes to these tests, so PR #653 already took care of it. Please merge those changes